pokemongo_bot/cell_workers/seen_fort_worker.py

```diff
@@ -74,15 +61,18 @@ def work(self):
             spin_details = response_dict['responses']['FORT_SEARCH']
             spin_result = spin_details.get('result', -1)
             if spin_result == 1:
-                logger.log("Loot: ", 'green')
                 experience_awarded = spin_details.get('experience_awarded',
                                                       False)
                 if experience_awarded:
+                    logger.log("Loot: ", 'green')
                     logger.log(str(experience_awarded) + " xp",
                                'green')
+                    if self.bot.softbanned == True:
```

Comment:
The only thing that should set us to *not* be softbanned anymore should be the softbanned worker.